src/engraving/dom/layoutbreak.cpp

```diff
@@ -238,6 +240,30 @@ String LayoutBreak::accessibleInfo() const
     return translatedSubtypeUserName();
 }
 
+//---------------------------------------------------------
+//   keySigs
+//---------------------------------------------------------
+
+std::vector<KeySig*> LayoutBreak::keySigs(bool all) const
```

Comment:
This shouldn't be a method of LayoutBreak. Better to put it in utils and perhaps call it something like keySigsAfterSectionBreak or something like that. Also, the variable `all` doesn't seem to be used by any called so let's remove it.